### PR TITLE
Add support for passing a resolver function when declaring a field

### PR DESCRIPTION
**Pull request by patrick91**
TODO: 

- [x] Check if it has correct annotations (return and arguments)
- [x] Support for arguments (now they are ignored)

### CODE REVIEW

**Comment by patrick91** [comment]
Going to do some refactoring while adding support for this, bear with me :)

File: strawberry/field.py [hunk 1]
```diff
@@ -103,19 +118,46 @@ class strawberry_field:
...
+        self.field_description = description
+        self.field_resolver = resolver
         self.is_subscription = is_subscription
-        self.description = kwargs.get("description", None)
-        self.name = kwargs.pop("name", None)
-        self.kwargs = kwargs
+        self.is_input = is_input
+
+        super().__init__(
+            # TODO:
```
Comment (marcoacierno) [inline strawberry/field.py]: What is this TODO left here?

Comment (patrick91) [reply strawberry/field.py]: I need to support default values when using field, forgot about it :D

**Comment by patrick91** [comment]
@marcoacierno I think I can merge this, do you want to have one final look? :)

**Pull request merged**